src/SmartComponents/PolicyRules/PolicyRules.js

```diff
@@ -1,13 +1,20 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { useQuery, gql } from '@apollo/client';
 import { useParams } from 'react-router-dom';
 import propTypes from 'prop-types';
-import PageHeader, {
-  PageHeaderTitle,
-} from '@redhat-cloud-services/frontend-components/PageHeader';
+import PageHeader from '@redhat-cloud-services/frontend-components/PageHeader';
 import RulesTable from '../../PresentationalComponents/RulesTable/RulesTable';
```

Comment:
let use relative import syntax:  
```suggestion
import RulesTable from '@/PresentationalComponents/RulesTable/RulesTable';
```

 there are other occurrences  below.